assets/js/form_helper.tsx

```diff
@@ -0,0 +1,62 @@
+export const UsStatesTerritories: [string, string][] = [
```

Comment:
There are 3 ways in the story suggested to implement the UsStateTerritories list. This is the third method. I think it is best to get PR merged in as is. I can come back and tinker with the other suggested methods so that I am not blocking anyone else.